src/Tempest/Database/src/IsDatabaseModel.php

```diff
@@ -54,7 +55,12 @@ public function getId(): Id
 
     public static function table(): TableName
     {
-        return new TableName(pathinfo(str_replace('\\', '/', static::class), PATHINFO_FILENAME));
+        $name = get(DatabaseConfig::class)
```

Comment:
Yeah I'm pretty sure it should be on `DatabaseConfig` and not the connection, unless you had a reason for it?